android/src/fdroid/play/listings/de-DE/release-notes.txt

```diff
@@ -1,11 +1,11 @@
-• Neue OSM-Daten vom 27. Februar
-• Div. Abstürze behoben
-• Fehlende Lokalisierung für chin. Langzeichen behoben
-• Dialoge für Hilfe und ausgewählte POI verbessert
-• Fehler bei komplexeren Öffnungszeiten behoben
-• Wiki-Artikel sind nun klickbar und für Lesezeichen sichtbar
-• Suche korrigiert und verbessert
-• Spezialisten im Gesundheitswesen hinzugefügt
-• Funktürme, Solar-, Wind-, Gas- und Wassergeneratoren hinzugefügt
-• Verbesserte ar-, nb-, pt- und tr-Übersetzungen
-• ...mehr: omaps.app/news
+• New OSM Planet data as of March 26
+• Experimental Lane assistant
+• Fix wrong location arrow position during navigation
+• Fix bookmark's color chooser for Right-To-Left languages
+• Properly display speed and distance in navigation mode
+• Fix API to select a point on the map
+• Many new shop types, including cannabis one :)
+• Add money transfer, bicycle charging stations, power plant
+• Made park, forest, and scrub colors different
+• Improved ar, be, de, hu, tr translations
+…more: omaps.app/news
```

Comment:
```suggestion
• Neue OSM Kartendaten vom 26. März
• Experimenteller Fahrspurenassistent
• Fehler bei Positionierung des Standortpfeiles bei der Navigation behoben
• Fehler bei der Darstellung des Farbwählers der Lesezeichen für Rechts-nach-Links Sprachen 
• Korrekte Darstellung von Geschwindigkeit und Distanz im Navigationsmodus
• Fehlerbehebung bei der API zum Auswählen eines Punktes auf der Karte
• Viele neue Geschäfte wurden hinzugefügt, unter anderem Kannabisläden ;)
• Neu hinzugefügte Objekte: Geldtransfer, Fahrradladestationen, Kraftwerk
• Die Farben von Parks, Wäldern und Buschland sind jetzt unterschiedlich
• Verbesserung der deutschen, türkischen, belarussischen und ungarischen Übersetzung

Bleib auf dem aktuellsten Stand unter: omaps.app/news
```

Reply:
It's 747 chars, for Android it should be no more than 500 chars. Can you please remove some sentences by editing your suggestion?

Reply:
Updated!